client/dive-common/components/Viewer.vue

```diff
@@ -381,12 +414,30 @@ export default defineComponent({
 
     watch(datasetId, reloadAnnotations);
 
-    const changeCamera = async (camera: string) => {
-      if (!camera || !baseMulticamDatasetId.value) {
-        throw new Error('Attempted to change camera to invalid value or baseMultiCamDatasetId was missing');
+    const controlsRef = ref();
+    const controlsHeight = ref(0);
+    const controlsCollapsed = ref(false);
+    function handleResize() {
+      if (controlsRef.value) {
+        controlsHeight.value = controlsRef.value.$el.clientHeight;
+        onResize();
       }
-      const newId = `${baseMulticamDatasetId.value}/${camera}`;
-      ctx.emit('update:id', newId);
+    }
+    const observer = new ResizeObserver(handleResize);
+    watch(controlsRef, (previous) => {
+      if (previous) observer.unobserve(previous.$el);
+      if (controlsRef.value) observer.observe(controlsRef.value.$el);
+    });
```

Comment:
Can you add a comment about what causes this watch to fire?  I think it's shadowing the value of the wrapping `v-if="progress.loaded"`, and I'm fine with doing it like this, I just want to make sure that's what's happening.